packages/api/src/app.ts

```diff
@@ -17,7 +17,7 @@ export const bootstrap = async () => {
     await umzug.down();
     process.exit(1);
   }
-  await sequelize.sync();
+  await sequelize.sync({ force: true });
```

Comment:
This will create all the necessary tables. Perfect for development. 
❗️When in production, make sure this is turned off, otherwise it will wipe the database.